_about/notes/index.md

```diff
@@ -16,17 +16,19 @@ toc: false
 
 {% include section-index.html docs=site.about %}
 
+The [latest](https://github.com/istio/istio/releases) Istio monthly release is {{site.data.istio.version}} which
+you can download with:
 
-- The [latest](https://github.com/istio/istio/releases) Istio monthly release is {{site.data.istio.version}}. It is downloaded when the following is used(*):
-  ```
-  curl -L https://git.io/getLatestIstio | sh -
-  ```
+```
+curl -L https://git.io/getLatestIstio | sh -
+```
 
-- The most recent 'stable' release is [0.2.12](https://github.com/istio/istio/releases/tag/0.2.12), the matching docs are [archive.istio.io/v0.2/docs/](https://archive.istio.io/v0.2/docs/)
-  ```
-  curl -L https://git.io/getIstio | sh -
-  ```
+The most recent stable release is [0.2.12](https://github.com/istio/istio/releases/tag/0.2.12). You can
+find the docs for that release [here](https://archive.istio.io/v0.2/). You can download 0.2.12 with:
 
-We typically wait to 'bake' the latest release for several weeks and ensure it is more stable than the previous one before promoting it to stable.
+```
+curl -L https://git.io/getIstio | sh -
+```
 
-> (*) Note: security conscious users should examine the output of the curl command before piping it to a shell.
+We typically wait to bake the latest release for several weeks and ensure it is more reliable than the previous one before
+promoting it to stable.
```

Comment:
Is this still accurate?  (re promoting it to stable...).  I think we only promote LTS to stable.

Reply:
it was purposely vague about using stable instead of LTS because LTS is quite a misnomer pre-1.0 (we are not committing to 1 year or any sort of long term support for those quarterly/(more) stable releases).  On the other hand I suppose if we defined "LTS" somewhere else on the site we can use that term.

Reply:
I removed the paragraph completely to avoid the whole issue.

WDYT?